#BE-4193 Release Notes Aug 5 (#588)
appcircleio/appcircle-docusaurus: docs/release-notes.md

# Latest Release Notes

## 3.20.1 - 2024-08-05 - Role Management Updates, Enterprise App Store and Publish Improvements, Bug Fixes and more

### 🆕 New Features

- [The Publish Environment Variables](/account/my-organization/role-management#publish-environment-variables) Module has been added within the role management with Manager and Viewer roles. <AccountBadge/> <CloudBadge/> <SelfHostedBadge/>
- [Integrations and Connection Management](/account/my-organization/role-management#integrations-and-connection-managements) roles have been added within the role management with Manager and Viewer roles. <AccountBadge/> <CloudBadge/> <SelfHostedBadge/>
- The Viewer role has been added for [Organization Management](/account/my-organization/role-management#organization-management-permissions). <AccountBadge/> <CloudBadge/> <SelfHostedBadge/>
- Hyperlink support has been added to the affected build profiles on the pop-up screen when a user attempts to delete a provisioning profile that is active in a build configuration. <SigningIdentitiesBadge/> <CloudBadge/> <SelfHostedBadge/>

> - The self-hosted Appcircle server script now includes a new command (`init`) that should be used after the export step once while installing the server, so that vault initialization is done seamlessly. <InfrastructureBadge/> <SelfHostedBadge/>

Reviewer: ```suggestion
- The self-hosted Appcircle server script now includes a new command (init) that should be used after the export step once while installing the server, ensuring seamless vault initialization. <InfrastructureBadge/> <SelfHostedBadge/>
```
